Userland/Libraries/LibGUI/Widget.cpp

```diff
@@ -1145,26 +1145,31 @@ void Widget::set_override_cursor(AK::Variant<Gfx::StandardCursor, NonnullRefPtr<
     }
 }
 
-bool Widget::load_from_gml(StringView gml_string)
+ErrorOr<void> Widget::try_load_from_gml(StringView gml_string)
 {
-    return load_from_gml(gml_string, [](DeprecatedString const& class_name) -> RefPtr<Core::Object> {
+    return try_load_from_gml(gml_string, [](DeprecatedString const& class_name) -> RefPtr<Core::Object> {
```

Comment:
Hmm can `unregistered_child_handler` also return an `Error`? Or is that super pervasive?

Reply:
sounds like a good idea, as far as I know the main users are this default handler and the GML playground placeholder generator. So we can interrupt the loading process if this fails and continue if not.

Reply:
Sure, makes sense.